code/modules/clothing/suits/utility.dm

```diff
@@ -134,16 +140,22 @@
 	resistance_flags = NONE
 	flags_2 = RAD_PROTECT_CONTENTS_2
 	sprite_sheets = list(
-		"Vox" = 'icons/mob/clothing/species/vox/head.dmi',
-		"Grey" = 'icons/mob/clothing/species/grey/head.dmi',
-		"Vulpkanin" = 'icons/mob/clothing/species/vulpkanin/helmet.dmi'
-		)
+		"Vox" = 'icons/mob/clothing/species/vox/head/utility.dmi',
+		"Unathi" = 'icons/mob/clothing/species/unathi/head/utility.dmi',
+		"Tajaran" = 'icons/mob/clothing/species/tajaran/head/utility.dmi',
+		"Vulpkanin" = 'icons/mob/clothing/species/vulpkanin/head/utility.dmi',
+		"Grey" = 'icons/mob/clothing/species/grey/head/utility.dmi',
+		"Drask" = 'icons/mob/clothing/species/drask/head/utility.dmi',
+		"Kidan" = 'icons/mob/clothing/species/kidan/head/utility.dmi'
+	)
 
 /obj/item/clothing/suit/radiation
 	name = "radiation suit"
 	desc = "A suit that protects against radiation. Label: Made with lead, do not eat insulation."
-	icon_state = "rad"
-	item_state = "rad_suit"
+	icon = 'icons/obj/clothing/suits/utility.dmi'
+	icon_override = 'icons/mob/clothing/suits/utility.dmi'
+	icon_state = "radsuit"
+	item_state = "radsuit"
```

Comment:
```suggestion
	item_state = 'icons/mob/clothing/suits/utility.dmi'
```

Reply:
This would break the sprite, but it also currently doesn't have the inhand working, so I made some.